themes/mlibrary_new/exhibit-builder/exhibits/show.php

```diff
@@ -66,7 +66,10 @@
       <?php $children = $exhibit_page->getChildPages();
                if ($children) {
                  foreach ($children as $child) {?>
-                   <h3> <?php echo metadata($child, 'title');?></h3>
+                   <header>
```

Comment:
Does this put multiple sibling `<header>` tags in the same parent element?  I don't think that's how `<header>` is supposed to work.

Reply:
You can have multiple Header element parts of semantic in the content.
https://www.w3schools.com/html/html5_semantic_elements.asp
What missing is adding an article or section outside around it. So the point is to make a subsection with header and content inside it, like <section><header><h2><p>..</p>/h2></header></section>.  We are still styling this page. it did not finish yet.

Reply:
Styling doesn't matter, I'm talking about the structure.  Yes, an html document can have multiple headers in it, but a single element cannot have multiple headers directly in it.  For example, the code here will create:
```html
<section>
  <div>
    <header>
      <h3>TITLE 1</h3>
    </header>
    <div>RENDER_PAGE 1</div>
    <header>
      <h3>TITLE 2</h3>
    </header>
    <div>RENDER_PAGE 2</div>
    <header>
      <h3>TITLE 3</h3>
    </header>
    <div>RENDER_PAGE 3</div>
  </div>
  ...
</section>
```
If you're telling me that these aren't done and are going to be wrapped in article tags later, that's a good direction to take it, but since it's not done, why are you merging it into trunk?   And why are you having me review it?

Reply:
We merge into the truck to be pulled by the UI developer to work on it. We can talk about different work follow approach going forward.